pandas/core/window/rolling.py

```diff
@@ -305,17 +290,6 @@ def _wrap_result(self, result, block=None, obj=None):
 
         if isinstance(result, np.ndarray):
 
-            # coerce if necessary
-            if block is not None:
-                if is_timedelta64_dtype(block.values.dtype):
```

Comment:
is this not reachable?

Reply:
Don't think so. Our test suite includes timedelta64 which doesn't his this branch.